homeassistant/components/airtouch4/strings.json

```diff
@@ -0,0 +1,16 @@
+{
+  "config": {
+    "error": {
```

Comment:
We're missing an abort string for `"already_configured"`. It will be used by `_async_abort_entries_match`.

Reply:
Ah, was wondering exactly how that worked! something like this right? https://github.com/home-assistant/core/pull/43513/commits/c28f075093bc5477db570c42662def068da645d1